CMakeLists.txt

```diff
@@ -21,7 +21,7 @@ set(PACKAGE_BUGREPORT "https://github.com/googleapis/google-cloud-cpp/issues")
 
 project(
     google-cloud-cpp
-    VERSION 2.23.0
+    VERSION 2.24.0
     LANGUAGES CXX)
 set(PROJECT_VERSION_PRE_RELEASE "")
```

Comment:
"rc"

Reply:
Done. Thanks. The merge commit overwrote it.

Reply:
Add it to both places (or run a build that generates `version_info.h`).

> The merge commit overwrote it.

Yikes.

Reply:
Done. Committed to fast.